decidim-core/app/controllers/concerns/decidim/with_participatory_processes.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module Decidim
+  # This module, when injected into a controller, adds two helper methods
+  # to fetch participatory processes & promoted participatory processes.
+  module WithParticipatoryProcesses
```

Comment:
I'd rather factor this duplication out in the form of `Rectify::Query`. `helper` methods can be defined per-controller, wouldn't worry about cosmetic duplication.